### PR TITLE
Unify commands

### PR DESCRIPTION
**Pull request by josefkarasek**
## Unify Commands

* Use updated commands from [#867](https://github.com/kubeshop/botkube/pull/867)
* Remove screen shots of that we no longer find valuable

## Related issue(s)

https://github.com/kubeshop/botkube/issues/703

### CODE REVIEW

File: docs/usage/event-notifications.md [hunk 1]
```diff
@@ -37,50 +37,48 @@ You can change the format by setting the `format` field in the [communication co
...
 To see how to update the configuration, see the [Updating the configuration](../configuration/index.md#updating-the-configuration) section in the Configuration document.
 
 ### Change notification sources
 
-To change the notification sources, you can either run the `helm upgrade` as described in [**View Botkube configuration**](#view-botkube-configuration) or run **@Botkube edit SourceBindings**.
+To change the notification sources, you can either run the `helm upgrade` command as described in [**View Botkube configuration**](#view-botkube-configuration) or run `@Botkube edit SourceBindings`.
 
 Depending on the communication platform, the possible options are being printed in response message:
 
+<!-- Is this screen shot valuable? -->
```
Comment (josefkarasek) [inline docs/usage/event-notifications.md:54]: 1. Is this screen shot valuable?

File: docs/usage/event-notifications.md [hunk 2]
```diff
@@ -37,50 +37,48 @@ You can change the format by setting the `format` field in the [communication co
...
+
 ![discord-edit-sources](assets/discord-edit-bindings.png)
 
 or you are able to pick them from an interactive list:
 
+<!-- Is this screen shot valuable? -->
+
 ![slack-edit-sources](assets/slack-edit-bindings.png)
```
Comment (josefkarasek) [inline docs/usage/event-notifications.md:62]: 2. Is this screen shot valuable?

File: docs/usage/event-notifications.md [hunk 3]
```diff
@@ -37,50 +37,48 @@ You can change the format by setting the `format` field in the [communication co
...
 ### Change notification sources
 
-To change the notification sources, you can either run the `helm upgrade` as described in [**View Botkube configuration**](#view-botkube-configuration) or run **@Botkube edit SourceBindings**.
+To change the notification sources, you can either run the `helm upgrade` command as described in [**View Botkube configuration**](#view-botkube-configuration) or run `@Botkube edit SourceBindings`.
 
 Depending on the communication platform, the possible options are being printed in response message:
 
+<!-- Is this screen shot valuable? -->
+
 ![discord-edit-sources](assets/discord-edit-bindings.png)
```
Comment (josefkarasek) [inline docs/usage/event-notifications.md:56]: 1. Is this screen shot valuable?

File: docs/usage/event-notifications.md [hunk 4]
```diff
@@ -37,50 +37,48 @@ You can change the format by setting the `format` field in the [communication co
...
+<!-- Is this screen shot valuable? -->
+
 ![slack-edit-sources](assets/slack-edit-bindings.png)
 
 When you save the new notification sources, changes are applied once the Botkube is restarted. It is an automated process which usually takes a few seconds.
 
 ### Disable notifications
 
-If you want to stop receiving notifications from Botkube, run **@Botkube notifier stop** from the configured channel where Botkube is added. You will no longer receive notifications from the Botkube in a given communication platform.
+If you want to stop receiving notifications from Botkube, run `@Botkube disable notifications` from the configured channel where Botkube is added. You will no longer receive notifications from the Botkube in a given communication platform.
```
Comment (mszostok) [inline docs/usage/event-notifications.md:68]: as you are adjusting the `**` to ``` please also update the @Botkube ping commands in Discord, Mattermost, and Slack pages 🙂

File: docs/usage/filters/index.md [hunk 5]
```diff
@@ -15,26 +15,20 @@ Botkube allows you to manage filters using **@Botkube** commands.
 
 ### List available filters
 
-Run **@Botkube filters list** to get list of available filters and their running status:
```
Comment (mszostok) [inline docs/usage/filters/index.md:18]: 2 lines above please change the `**@Botkube**` to backticks

**Review by mszostok — APPROVED** [review]
please address minor comments before merging 👍

**Review by lpetkov — APPROVED** [review]
Ship it!

**Pull request merged**